### PR TITLE
[WIP ]Making line corrections and updating copy-right

### PR DESCRIPTION
**Pull request by Arafatk**
Please review this.If this seems correct to you,then i will make change for the whole lib/nmatrix directory.
Thanks

### CODE REVIEW

File: lib/nmatrix/yale_functions.rb [hunk 1]
```diff
@@ -89,7 +93,8 @@ def yale_ja_d_keys_at i
   # call-seq:
   #     yale_ja_d_keys_set_at(i) -> Set
   #
-  # Returns the diagonal and non-diagonal column indices stored in a given row.
+  # Returns the diagonal and non-diagonal column
+  # indices stored in a given row.
```
Comment (v0dro) [inline lib/nmatrix/yale_functions.rb]: How about we indent multi-line comments with two spaces? That makes it obvious that the next line is a continuation of the previous.

**Comment by v0dro** [comment]
Looks good. Let's just be sure that nothing has broken as a result of the line breaks.

**Comment by Arafatk** [comment]
@v0dro  That seems better.

File: lib/nmatrix/yale_functions.rb [hunk 2]
```diff
@@ -70,7 +72,8 @@ def yale_ja_sorted_set_at i
   # call-seq:
   #     yale_nd_row_as_hash(i) -> Hash
   #
-  # Returns the non-diagonal column indices and entries stored in a given row.
+  # Returns the non-diagonal column indices
+  # and entries stored in a given row.
```
Comment (v0dro) [inline lib/nmatrix/yale_functions.rb]: What I meant to say, is that the above lines should look like this:

``` ruby
# Returns the non-diagonal column indices
#   and entries stored in a given row.
```

**Comment by mohawkjohn** [comment]
Is this still a work-in-progress?

**Comment by Arafatk** [comment]
@mohawkjohn I am sorry,but i have my mid-term exams. I will restart working from upcoming Tuesday.

**Comment by Arafatk** [comment]
@v0dro @mohawkjohn Can you give a few suggestions for the possible changes you wish to see? 
As of now I I have completed almost all the files in lib/nmatrix directory.
Their are merge conflicts in some files and will resolve them.

**Comment by Arafatk** [comment]
@mohawkjohn I will revert line splits of that kind. 
I am still confused as to how the spec tests were passing after making that change ?

**Comment by Arafatk** [comment]
@mohawkjohn @v0dro Should i go with multi-line strings divided by `\` or should i leave the strings as it is?

**Comment by v0dro** [comment]
@mohawkjohn wants the strings the way they are. Let them be. But be sure to change the rest of the code.

**Comment by v0dro** [comment]
Sure! Or you can rebase this one.

**Comment by v0dro** [comment]
Can we merge this @mohawkjohn ?

**Comment by mohawkjohn** [comment]
@v0dro I think @Arafatk owes us a rebase first.

**Comment by v0dro** [comment]
Directly squash with github?

**Pull request merged**